static/app/views/starfish/components/datePicker.tsx

```diff
@@ -0,0 +1,21 @@
+import DatePageFilter from 'sentry/components/datePageFilter';
+import {t} from 'sentry/locale';
+
+function StarfishDatePicker() {
+  return (
+    <DatePageFilter
+      maxDateRange={7}
+      disallowArbitraryRelativeRanges
+      relativeOptions={{
+        '1h': t('Last hour'),
+        '12h': t('Last 6 hours'),
+        '24h': t('Last 24 hours'),
+        '3d': t('Last 3 days'),
+        '7d': t('Last 7 days'),
+      }}
```

Comment:
This has a side effect that typing in something like `2d` no longer works because it's not in this list. I think you can pass a _function_ here instead, and filter out anything longer than 7 days. There are some examples in the codebase!

Reply:
Thanks! Didn't know that I'll update this accordingly